tox.ini

```diff
@@ -184,6 +184,31 @@ commands_pre=
 commands=
     yea {env:CI_PYTEST_SPLIT_ARGS:} --strict --shard {env:SHARD:default} run {posargs:--all}
 
+[testenv:reports-py{38,39,310,311,312}]
```

Comment:
i don't think you need a new table, you probably can add it to `[testenv:{,core-}{,notebooks-}py{37,38,39,310,311,312}]` just add the relevant  labels

Reply:
[testenv:{,core-}{,notebooks-, reports-}py{37,38,39,310,311,312}]